pkg/controller/networkpolicy/store/networkpolicy.go

```diff
@@ -62,6 +62,9 @@ func (event *networkPolicyEvent) ToWatchEvent(selectors *storage.Selectors, isIn
 	case !currObjSelected && !prevObjSelected:
 		return nil
 	case currObjSelected && !prevObjSelected:
+		if event.CurrObject.RealizableMessage != "" {
+			return nil
+		}
```

Comment:
adding here is not enough if we still calculate node span for unrealizable policy because the policy could be sent to agents when `currObjSelected` and `prevObjSelected` are true, which may mess up agents' cache. But if the above suggestion is adopted, we can just remove the code here.

Reply:
Implemented the above method, cleared span meta before sending to agent.